### PR TITLE
SElinux policy for Foreman core

### PR DESCRIPTION
**Pull request by skottler**

### CODE REVIEW

File: selinux/foreman.fc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+foreman		                 --	gen_context(system_u:object_r:httpd_foreman_script_exec_t,s0)
```
Comment (xsuchy) [inline selinux/foreman.fc:1]: Should be here full path?

File: selinux/foreman.fc [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+/var/log/foreman(/.*)?                  gen_context(system_u:object_r:httpd_foreman_script_log_t,s0)
+/var/lib/foreman/db                     gen_context(system_u:object_r:var_lib_t,s0)
+/var/lib/foreman/db(/.*.sqlite3)?       gen_context(system_u:object_r:foreman_db_t,s0)
+/usr/share/foreman(/.*)?                gen_context(system_u:object_r:httpd_foreman_script_exec_t,s0)
+/etc/foreman(/.*)?                      gen_context(system_u:object_r:foreman_config_t,s0)
+/etc/logrotate.d/foreman                gen_context(system_u:object_r:logrotate_exec_t,s0)
```
Comment (xsuchy) [inline selinux/foreman.fc:7]: Why this one is needed. Ordinary logrotate config have system_u:object_r:etc_t:s0 attribute.

Comment (skottler) [inline selinux/foreman.fc:7]: Yeah, looking at this again it should probably just be `logrotate_t`. Most of the things labelled as `logrotate_exec_t` are the actual executables in `/usr/sbin` or `/usr/bin`.

File: selinux/foreman.fc (hunk 1 above)
Comment (skottler) [inline selinux/foreman.fc:1]: Nope, this is just the generic context. Same thing as here - https://github.com/Katello/katello/blob/master/selinux/katello-selinux/katello.fc#L14.

**Comment by ohadlevy** [comment]
this should be moved to the packaging repo instead, as this is per OS anyway

**Comment by xsuchy** [comment]
Ohad, generaly I agree with you.
But in this specific case I do not see any OS specific part here.

**Comment by skottler** [comment]
@xsuchy his point is that this doesn't apply to all the platforms we support so it probably makes more sense to be part of a the packaging stuff where things are inherently OS-specific.

**Comment by xsuchy** [comment]
@skottler I understand that Foreman support more OSes. But I think this policy will look exactly the same for Debian, Fedora and RHEL. Which part would be different for some OS? All supported OS (AFAIK) use /var/log, /var/lib/, /etc/logrotate.d etc.

**Comment by skottler** [comment]
Right, I understand where you're coming from, but it seems unlikely we'll ever ship a package for Debian so this is going to be RHEL-derivative specific.

**Comment by skottler** [comment]
Also, I've opened theforeman/foreman-rpms#51.

**Pull request closed without merging**